mne/epochs.py

```diff
@@ -662,7 +662,7 @@ def __next__(self, *args, **kwargs):
     def __hash__(self):
         if not self.preload:
             raise RuntimeError('Cannot hash epochs unless preloaded')
-        return object_hash(dict(info=self.info, data=self._data))
+        return object_hash(dict(info=hash(self.info), data=self._data))
```

Comment:
turns out to be a bit problematic. In Python3, every object has a buildin `__hash__` function, but this default hash function is not compatible with `object_hash`.

A solution could be to add an `@object_has_compatible` decorator to custom hash functions and check for that. But maybe its overkill in this case.